configs/pontoon.toml

```diff
@@ -327,3 +327,9 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/platform-post-download.ftl"
     l10n = "{locale}/products/vpn/platform-post-download.ftl"
+[[paths]]
```

Comment:
Let's move this entry under the en/mozorg/... section, so they are in alphabetical order.